bookletimposerwrapper.sh

```diff
@@ -0,0 +1 @@
+cd /app/share && bookletimposer
```

Comment:
You could also set `command` to `/app/share/bookletimposer` unless it requires working directory to be `/app/share`.

Reply:
I don't know why but the app don't work if  its not in app/share that why i use it :/ if you know why im interesed to patch it

Reply:
I try this but it don't work don't ```flatpak-builder --run build-dir org.codecoop.kjo.bookletimposer.yaml /app/share/bookletimposer     

(flatpak-builder:9064): flatpak-builder-WARNING **: 19:08:50.739: Unknown property version for type BuilderManifest
bwrap: execvp /app/share/bookletimposer: Permission denied
```

Reply:
It probably misses executable bit. `chmod +x` should fix it.